run.py

```diff
@@ -1,7 +1,14 @@
+import eventlet
+eventlet.monkey_patch()  # required for server to run properly
+
 from app import app, socketio
+from app.devices import measure_colors, sensor, led
+
+
+if __name__ == '__main__':
+    # run the sensor and led control in a separate coroutine
+    eventlet.spawn(measure_colors, socketio, sensor, led)
```

Comment:
Wydaje mi się że `eventlet.spawn(measure_colors, socketio, sensor, led)` powinno być przed `if`'em bo nie zawsze skrypt musi być uruchamiany prosto z interpretera. Gdyby przed `Flask`'iem stał powiedzmy `Gunicorn` to nie załapie się na wykonanie tej linii.